spek-ide-plugin/intellij-idea/src/main/resources/META-INF/plugin.xml

```diff
@@ -0,0 +1,40 @@
+<idea-plugin>
+    <id>org.spekframework</id>
+    <name>Spek</name>
+    <vendor email="team@spekframework.org" url="http://spekframework.org">Spek Team</vendor>
+    <description><![CDATA[
+      Official IntelliJ IDEA plugin for <a href="http://jetbrains.github.io/spek/">Spek</a>.
+    ]]></description>
+
+    <change-notes><![CDATA[
+      TBD
+    ]]>
+    </change-notes>
+
+    <!-- please see http://www.jetbrains.org/intellij/sdk/docs/basics/getting_started/build_number_ranges.html for description -->
+    <idea-version since-build="162.0"/>
+
+    <!-- please see http://www.jetbrains.org/intellij/sdk/docs/basics/getting_started/plugin_compatibility.html
+         on how to target different products -->
+    <!-- uncomment to enable plugin in all products
+    <depends>com.intellij.modules.lang</depends>
```

Comment:
Most likely you can remove similar comments.

Reply:
ack